### PR TITLE
Systemd files

### PR DESCRIPTION
**Pull request by artazar**
Adding systemd files for stub and db services
Correcting nginx sample file to avoid eternal restart loop

### CODE REVIEW

**Comment by okeer** [comment]
Can one of the admins verify this patch?

**Comment by artazar** [comment]
> Can one of the admins verify this patch?

Note that the image names are placeholders - they do not exist on docker hub repo yet to be pulled.

**Comment by marvin4ward** [comment]
test

File: docker/db/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Initialization
 
-To start the mysql docker container:
+To start the project1 mysql docker container:
 
-$ docker run --name project1-mysql -e MYSQL_ROOT_PASSWORD=1q2w3e -d mysql:latest
+$ docker run --name project1-mysql -e MYSQL_ROOT_PASSWORD=1q2w3e -d devops2019summer/db
```
Comment (marvin4ward) [inline docker/db/README.md]: Also `latest` ?

File: systemd/nginx.service [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+Description=project1 nginx service
 Requires=docker.service
 After=docker.service
 
 [Service]
 Restart=always
 RestartSec=3
 ExecStartPre=/bin/sh -c "/usr/bin/docker rm -f docker-nginx 2> /dev/null || /bin/true"
-ExecStart=/usr/bin/docker run --name docker-nginx -p 80:80 -d nginx
+ExecStart=/usr/bin/docker run --name project1-nginx -p 80:80 --rm nginx
```
Comment (marvin4ward) [inline systemd/nginx.service]: Nginx must use network from stub container.
but we can work with that so far

**Comment by okeer** [comment]
Sup, do something

**Comment by okeer** [comment]
Ok, need to rebase this and push it again - this should trigger PR webhook and we will be able to merge thsi PR.

**Comment by okeer** [comment]
test this please

**Comment by okeer** [comment]
ok to test

File: docker/db/README.md (hunk 1 above)
Comment (artazar) [reply docker/db/README.md]: correct, latest tag added, will be fixed in the next PR

File: systemd/nginx.service (hunk 2 above)
Comment (artazar) [reply systemd/nginx.service]: networking part will be addressed later

File: .gitignore [hunk 3]
```diff
@@ -1 +0,0 @@
-.vagrant
```
Comment (marvin4ward) [inline .gitignore]: Why this going to be deleted?

Comment (artazar) [reply .gitignore]: Fixed

**Pull request merged**